tests/Interactive/InstantBookTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Interactive;
+
+use App\InteractiveSlide\InstantBook;
+use Doctrine\ORM\EntityManager;
+use Hautelook\AliceBundle\PhpUnit\BaseDatabaseTrait;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+class InstantBookTest extends KernelTestCase
+{
+    use BaseDatabaseTrait;
+
+    private EntityManager $entityManager;
+    private ContainerInterface $container;
+
+    public static function setUpBeforeClass(): void
+    {
+        static::bootKernel();
+        static::ensureKernelTestCase();
+        if (!filter_var(getenv('API_TEST_CASE_DO_NOT_POPULATE_DATABASE'), FILTER_VALIDATE_BOOL)) {
+            static::populateDatabase();
+        }
+    }
+
+    public function setUp(): void
+    {
+        $this->container = static::getContainer();
+        $this->entityManager = $this->container->get('doctrine')->getManager();
+    }
+
+    public function testIntervalFree(): void
+    {
+        $service = $this->container->get(InstantBook::class);
+
+        $schedules = [
+            [
+                'startTime' => (new \DateTime())->add(new \DateInterval('PT30M')),
```

Comment:
```suggestion
                'startTime' => new \DateTime('+30 min'),
```

is much easier to read – and write.